src/dbos-executor.ts

```diff
@@ -973,6 +1294,7 @@ export class DBOSExecutor implements DBOSExecutorContext {
     }
     await this.scheduler?.destroyScheduler();
     wfQueueRunner.stop();
+    await this.wfqEnded;
```

Comment:
That was a pre-existing bug right?

Reply:
I think the accounting just moved; had been in the runtime.